src/controller/backend/Program.cs

```diff
@@ -67,8 +69,15 @@
         policy.AddAuthenticationSchemes(JwtBearerDefaults.AuthenticationScheme);
         policy.RequireAuthenticatedUser();
     });
+    options.AddPolicy("Permission", policy =>
+    {
+        policy.Requirements.Add(new PermissionRequirement(""));
```

Comment:
What does a empty permission mean in this system?

Reply:
just let it be for now :3